instance.go

```diff
@@ -982,6 +982,11 @@ func (instance *Instance) querySchemaTables(schema string) ([]*Table, error) {
 			if flavor.HasDataDictionary() && len(t.SecondaryIndexes) > 1 {
 				fixIndexOrder(t)
 			}
+			// Foreign keys order is unpredictable in MySQL before 5.6, so reorder
+			// foreign keys based on parsing SHOW CREATE TABLE if needed
+			if !flavor.MySQLishMinVersion(5, 6) && len(t.ForeignKeys) > 1 {
```

Comment:
Minor nit, this will accidentally be triggered for MariaDB since MySQLishMinVersion only ever returns true for MySQL + Percona Server. No need to hold up the PR for this though, I'll tweak in a followup commit.